test/runtests.jl

```diff
@@ -396,3 +396,33 @@ end
         @test f(x) ≈ x^2
     end
 end
+
+@testset "Half measures" begin
+    @testset "HalfNormal" begin
+        d = Normal(σ=3)
+        h = HalfNormal(3)
+        x = rand(h)
+        @test density(h, Lebesgue(ℝ), x) ≈ 2 * density(d, Lebesgue(ℝ), x)
+    end
+
+    @testset "HalfCauchy" begin
+        d = Cauchy(σ=3)
+        h = HalfCauchy(3)
+        x = rand(h)
+        @test density(h, Lebesgue(ℝ), x) ≈ 2 * density(d, Lebesgue(ℝ), x)
+    end
+
+    @testset "HalfLaplace" begin
```

Comment:
Isn't the half-Laplace just the Exponential distribution?

Reply:
Oh weird, hadn't ever thought of that. That sounds right, if it is we can just drop `HalfLaplace`